search/collectors/topn_test.go

```diff
@@ -398,21 +399,28 @@ func TestPaginationSameScores(t *testing.T) {
 			t.Errorf("doc ID %s is in top 5 and next 5 result sets", hit.ID)
 		}
 	}
-
 }
 
 func BenchmarkTop10of100000Scores(b *testing.B) {
-	benchHelper(10000, NewTopScorerCollector(10), b)
+	benchHelper(10000, func() search.Collector {
```

Comment:
The count mismatches were noted earlier in the thread.  Initially I didn't want to change them here because it was going to complicate the merge.  Since then I've already complicated the merge.  So yes I will correct this either during the merge, or in a commit after the merge.